spec/imsc-hrm.html

```diff
@@ -703,10 +858,12 @@ <h2>Paint Text</h2>
         </tbody>
       </table>
 
-      <p class='note'>The choice of font by the presentation processor can increase rendering complexity.
-      For instance, a cursive font might select different glyphs for a given <a data-cite="i18n-glossary/#def_grapheme">grapheme</a>
-      (in order to maintain joining or for the start/end of the word) even in the Latin script. Conversely the rendering of scripts that fall in the <i>any other value</i> category can in 
-      practice achieve performance comparable to, say, the <code>latin</code> script.</p>
+      <p class='note'>The choice of font by the <dfn data-cite="ttml2#terms-presentation processor">presentation
+      processor</dfn> can increase rendering complexity. For instance, a cursive font might select different glyphs for
```

Comment:
Might be worth clarifying that this potentially increased rendering complexity is _presentation time_ complexity rather than HRM-defined complexity?

Reply:
That's great, I'm being slightly baffled by what the GitHub UI is showing me on the web, because that commit doesn't show in the "New changes since you last viewed" even though it is apparently a commit on the same branch, but I'm going to assume it's all fine and resolve this conversation!